app/endpoints/words.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter
+from app.services.client import client
+
+router = APIRouter()
+
+@router.get("/getRandom")
+def getRandom():
+    APIclient = client(baseURL="https://random-words5.p.rapidapi.com", apiKey="9ef31d92a4msh85731ca38c62a57p1e7318jsn7ecaa55b0256")
```

Comment:
don't initiate the client in each endpoint, instead initiate it once in the file of the client and import the initiated client to use in the endpoints,

also export the credentials to environment variables from .env file which is not exported to git, and never upload credentials to git